### PR TITLE
Update input-method-editor-error.md

### PR DESCRIPTION
**Pull request by tyadon**
These issues were addressed in 9C and 10C of last year. CSS has requested that we add some additional details that the customer has requested to see officially documented.

### CODE REVIEW

**Comment by PRMerger7** [comment]
@tyadon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ ms.date: 10/06/2020
 ---
 # Input method editor issue in RemoteApp scenarios
 
-We've been made aware of the following issues with the input method editor (IME) through user feedback:
+The following issues have been addressed in Windows 10+ operating systems as of October 2021.
```
Comment (Heidilohr) [inline WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md:13]:
```suggestion
The following issues have been addressed in Windows 10 and later as of October 2021.

```

File: WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md [hunk 2]
```diff
@@ -19,6 +19,9 @@ We've been made aware of the following issues with the input method editor (IME)
 Affected platforms:
 
 - Azure Virtual Desktop
-- Remote Desktop Services (so far, this issue has only been reported for Windows Server 2019)
+- Remote Desktop Services
 
-We're currently working on a solution to help solve this issue.
+Please Note:
```
Comment (Heidilohr) [inline WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md:24]: Delete this line. We don't say "please" in documentation.

File: WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md [hunk 3]
```diff
@@ -19,6 +19,9 @@ We've been made aware of the following issues with the input method editor (IME)
 Affected platforms:
 
 - Azure Virtual Desktop
-- Remote Desktop Services (so far, this issue has only been reported for Windows Server 2019)
+- Remote Desktop Services
 
-We're currently working on a solution to help solve this issue.
+Please Note:
+
```
Comment (Heidilohr) [inline WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md:25]: >[!NOTE]

File: WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md [hunk 4]
```diff
@@ -19,6 +19,9 @@ We've been made aware of the following issues with the input method editor (IME)
 Affected platforms:
 
 - Azure Virtual Desktop
-- Remote Desktop Services (so far, this issue has only been reported for Windows Server 2019)
+- Remote Desktop Services
 
-We're currently working on a solution to help solve this issue.
+Please Note:
+
+- Windows 8.1 and below are out of service and remain affected
```
Comment (Heidilohr) [inline WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md:26]:
```suggestion
> - Windows 8.1 and earlier are out of service and remain affected.

```

File: WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md [hunk 5]
```diff
@@ -19,6 +19,9 @@ We've been made aware of the following issues with the input method editor (IME)
...
 
 - Azure Virtual Desktop
-- Remote Desktop Services (so far, this issue has only been reported for Windows Server 2019)
+- Remote Desktop Services
 
-We're currently working on a solution to help solve this issue.
+Please Note:
+
+- Windows 8.1 and below are out of service and remain affected
+- Fix must be applied on both the Remote Desktop Server and Client machines
```
Comment (Heidilohr) [inline WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md:27]:
```suggestion
- You must apply the update to both the Remote Desktop server and the client devices.

```

**Review by Heidilohr — CHANGES_REQUESTED** [review]

**Comment by PRMerger18** [comment]
@tyadon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md (hunk 2 above)
Comment (tyadon) [reply WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md:25]: Approved

File: WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md (hunk 3 above)
Comment (tyadon) [reply WindowsServerDocs/remote/remote-desktop-services/troubleshoot/input-method-editor-error.md:26]: Approved

**Comment by PRMerger15** [comment]
@tyadon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request closed without merging**